packages/core/src/external/fhir/export/string/resources/immunization.ts

```diff
@@ -102,6 +102,69 @@ export class ImmunizationToString implements FHIRResourceToString<Immunization>
       parts.push(performerStr);
     }
 
+    if (immunization.isSubpotent !== undefined) {
+      parts.push(`is subpotent: ${immunization.isSubpotent}`);
+    }
+
+    const programEligibilityStr = formatCodeableConcepts({
+      concepts: immunization.programEligibility,
+      label: "Program Eligibility",
+      isDebug,
+    });
+    if (programEligibilityStr) {
+      parts.push(programEligibilityStr);
+    }
+
+    const fundingSourceStr = formatCodeableConcept({
+      concept: immunization.fundingSource,
+      label: "Funding Source",
+      isDebug,
+    });
+    if (fundingSourceStr) {
+      parts.push(fundingSourceStr);
+    }
+
+    if (immunization.reaction) {
+      const reactions = immunization.reaction
+        .map(reaction => {
+          const components = [
+            reaction.date && (isDebug ? `Date: ${reaction.date}` : reaction.date),
+            formatCodeableConcept({ concept: reaction.detail, label: "Detail", isDebug }),
+            // reaction.reported !== undefined &&
```

Comment:
The pattern on these files has been to leave the properties we're not exposing (but could) commented out. Can review the approach later if needed.